src/types/Observations.ts

```diff
@@ -1,23 +1,45 @@
 import { components } from 'src/api/types/generated-schema';
+import { MultiPolygon } from './Tracking';
 
 // basic information on a single observation (excluding observation results)
 export type Observation = components['schemas']['ObservationPayload'];
 
 // "Upcoming" | "InProgress" | "Completed" | "Overdue"
 export type ObservationState = Observation['state'];
 
+type Boundary = {
+  boundary: MultiPolygon;
+};
+
 // expanded information on an observation including observed results down to monitoring plot level detail
 // requires navigating a tree of zone results -> subzone results -> ( species results | monitoring plot results )
-export type ObservationResults = components['schemas']['ObservationResultsPayload'];
+export type ObservationResultsPayload = components['schemas']['ObservationResultsPayload'];
+export type ObservationResults = ObservationResultsPayload &
```

Comment:
@charlie-tf I renamed the previous `ObservationResults` to `ObservationResultsPayload` (same as in the generated types). Extended it with a boundary and name. Similarly in zones and subzones.

Reply:
This had to change a bit.